src/signals/incident-management/containers/IncidentOverviewPage/components/List/index.js

```diff
@@ -14,6 +15,15 @@ class List extends React.Component { // eslint-disable-line react/prefer-statele
     }
   }
 
+  getExpiredDays(incident) {
```

Comment:
The function and variables names are not corresponding with what the are doing/meaning/representing. I would suggest something like:
getElapsedDays(incident)
const created/createdAt/start
const elapsed/elapsedTime